src/parser/parser.cpp

```diff
@@ -660,6 +661,12 @@ void ParserState::pushScope(bool isUserContext)
 void ParserState::pushGetValueScope()
 {
   pushScope();
+  // We cannot ask for the model domain elements if we are in parse-only mode.
+  // Hence, we do nothing here.
+  if (d_parseOnly)
+  {
+    return;
+  }
```

Comment:
Is `d_parseOnly` necessary? Wouldn't it be better to move the code below to `GetValueCommand::invoke`?

Reply:
No this isn't possible, unfortunately it's more complicated in this case.

The problem is that commands have access to the symbol manager, but not the parser.  

Moreover, the code here is run to *parse* the GetValueCommand, not when it is run.

Reply:
I see... I forgot that we're allowed to mention abstract values in `get-value` command. In that case, can we enable the dump tester for `test/regress/cli/regress0/parser/issue6908-get-value-uc.smt2` now?

Reply:
No, that issue still fails, since that regression requires binding the symbol for the uninterpreted sort value, which this check prevents.